python.mk

```diff
@@ -37,107 +40,132 @@ ${PYTHON} ${ENV_DIR} ${PIP}:
 	${PIP} install --upgrade pip==9.0.1
 	${PIP} install wheel==0.30.0
 
-# Dummy targets onto which the targets defined in python_component below are
-# added as dependencies.
-${ENV_DIR}/.wheels-cleaned:
 	touch $@
 
-${ENV_DIR}/.wheels-built:
-	touch $@
+# This target builds all required wheelhouses and is therefore expected to be a
+# dependency of `make deb`
+.PHONY: wheelhouses
+
+# A dummy test target, to which dependencies are added for each test component
+.PHONY: test
 
-${ENV_DIR}/.wheels-installed:
+# Common coverage target. Because coverage is run over all components at once,
+# COVERAGE_SETUP_PY must be specified as a list of setup.py files that are run
```

Comment:
Can we create this automatically by doing:
```
COVERAGE_SETUP_PY += $1_SETUP
```
in `python_component`?

Reply:
Not so simply, because we may have test targets that we don't include include in coverage (e.g. the clearwater-etcd fv_test target). Although maybe the solution is to remove that from the test target, and create that one manually